schema.sql

```diff
@@ -6,4 +6,38 @@ CREATE TABLE animals (
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
```

Comment:
Please be sure to add the new queries into your queries.sql file as it is part of the requirements 🤓

Reply:
Hello @mateo951, thanks for reviewed feedback. There is one thing I want to point out from the documentation, and I believe you see my perspective. Inasmuch as I would implement what you have asked, from the attached image, it was not explicitly stated we add the queries to the` queries.sql.` 
![feedback](https://user-images.githubusercontent.com/92553079/190501506-bf04d102-3649-489e-be6e-8be0ef969ddf.PNG)